modules/govuk_jenkins/templates/jobs/copy_data_from_production_to_aws.yaml.erb

```diff
@@ -24,15 +24,24 @@
         - build-discarder:
             days-to-keep: 30
             artifact-num-to-keep: 5
+        - slack:
+            notify-start: true
+            notify-success: true
+            notify-aborted: false
+            notify-notbuilt: false
+            notify-unstable: false
+            notify-failure: true
+            notify-backtonormal: false
+            notify-repeatedfailure: false
+            include-test-summary: false
+            room: <%= @slack_room %>
```

Comment:
As far as I can see, the Slack room is hardcoded in the `govuk_jenkins::jobs::copy_data_from_production_to_aws` class to `govuk-infrastructure`, which is archived. Does this need updating?

Reply:
hi @issyl0 , perhaps this need to go as a separate trello card and dealt separately.

Reply:
It should.

Reply:
https://trello.com/c/d6mKRbbN/62-data-sync-slack-notification

Reply:
Looks like this is being addressed here https://github.com/alphagov/govuk-puppet/pull/7476 now.